From d6d5e1ebca22ef5b399463246b15b9da1b500769 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Mon, 16 Oct 2017 09:17:11 +0800
Subject: [PATCH 2693/5242] MLK-16590 ARM64: dts: fsl-imx8qxp-mek: enable USB3
 and PTN5110

commit  b22988a0866ca693179d2b12f7ef459f6eb3a9f1 from
https://source.codeaurora.org/external/imx/linux-imx.git

All imx8qxp mek board will replace NTB0104 with NTS0104, so it is ok
to enable PTN5110 and USB3 by default, see below commit for detail:
commit 5989fe321b3026 ("MLK-16522-4 ARM64: dts: fsl-imx8qxp-mek:
add USB3 support")

BuildInfo:
- SCFW 1f59442e, IMX-MKIMAGE fb52c576, ATF
- U-Boot 2017.03-imx_v2017.03+g34be5a2

Acked-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 3d342c3..7d1cb42 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -647,7 +647,7 @@
 		op-snk-mw = <9000>;
 		port-type = "drp";
 		default-role = "sink";
-		status = "disabled";
+		status = "okay";
 	};
 };
 
@@ -675,7 +675,7 @@
 &usbotg3 {
 	dr_mode = "otg";
 	extcon = <&typec_ptn5110>;
-	status = "disabled";
+	status = "okay";
 };
 
 &usdhc1 {
-- 
1.7.9.5

